BUG 1429224 new section for htpasswd command

mburke5678 opened the pull request:
WIP

made edits based on QA review

new section for htpasswd command

new section for htpasswd command, per BZ-1429224

mburke5678 commented:
@ahardin-rh @adellape PTAL

ahardin-rh commented on admin_solutions/master_node_config.adoc:
```diff
@@ -130,12 +144,59 @@ $ ansible-playbook -b -i ./hosts ~/src/openshift-ansible/playbooks/byo/config.ym
+
+To configure the {product-title} cluster to use HTPasswd authentication, you need at least one user with a hashed password. Add the username and password directly in the *_./hosts_* inventory file or pass the credentials in a flat file, as xref:config-htpasswd[as shown above]. 
+
+To create a user and hashed password
```
Maybe format this as a procedure. 

To create a user and hashed password:

. Run:
+
----
$ htpasswd -n <user_name>
----

.  Enter and confirm . . .

ahardin-rh commented:
@mburke5678 just one comment from me. Looks good!

mburke5678 commented:
@ahardin-rh Can you take another look? I changed the section in question to a numbered procedure and made a few other changes due to the list (added links to that numbered list; changed the following paragraph to a procedure).

ahardin-rh commented on admin_solutions/master_node_config.adoc:
```diff
@@ -116,12 +122,26 @@ ec2-52-95-5-36.compute-1.amazonaws.com  openshift_ip=172.17.3.89 openshift_publi
++
+Because {product-title} requires a hashed password to configure HTPasswd authentication, you can use the `htpasswd` command, xref:htpasswd[as shown in the following section], to generate the hashed password(s) for your user(s) or to create the flat file with the users and associated hashed passwords.
++ 
+The following example, will change the authentication method from the default `deny all` setting to `htpasswd` and use the specified file to generate user IDs and passwords for the `jsmith` and `bloblaw` users.
```
Looks like an unnecessary comma after example

mburke5678 replied:
+1

ahardin-rh commented:
@mburke5678 Just one comment from me. Otherwise, looks good!

mburke5678 commented:
[rev_history]
|xref:../install_config/configuring_authentication.adoc#install-config-configuring-authentication[Installation and Configuration -> Configuring Authentication and User Agent]|Added information on using the `htpasswd` command to generate hashed passwords for HTPasswd authentication to the xref:../install_config/configuring_authentication.adoc#HTPasswdPasswordIdentityProvider[HTPasswd] section.
%
|xref:../admin_solutions/master_node_config.adoc#admin-solutions-master-node-config[Administrator Solutions -> Master and Node Configuration]|Added information on using the `htpasswd` command to generate hashed passwords for HTPasswd authentication to the xref:../admin_solutions/master_node_config.adoc#master-node-config-ansible[Making Configuration Changes Using Ansible] section.
%
